metadata.rb

```diff
@@ -11,7 +11,7 @@
 supports         'centos', '~> 7.0'
 supports         'centos_stream', '~> 8.0'
 
-depends          'composer'
+depends          'composer', '~> 3.0.0'
```

Comment:
Let's loosen this up a bit.
```suggestion
depends          'composer', '~> 3.0'
```